src/subcommands/decrypt.rs

```diff
@@ -24,7 +24,7 @@ pub struct Decrypt {
 impl Decrypt {
     /// Decrypts data.
     pub fn run(&self, basic_client: BasicClient) -> Result<()> {
-        let input = base64::decode(self.input_data.as_bytes().to_vec())?;
```

Comment:
Just curious as to what prompted this change? We're not consuming rcgen at this point. Is that just a bit of unrelated clean-up?